### PR TITLE
[HDF5] python module

### PR DESCRIPTION
**Pull request by philbucher**
using the new python import system

note that after this the python-scripts have to be imported with a preceeding KratosMultiphysics.HDF5Application

If it is used through the process-factory then no changes are necessary

This is the new standard

### CODE REVIEW

**Review by sunethwarna — DISMISSED** [review]

**Comment by philbucher** [comment]
I will postpone merging this until #4627 is solved, since otherwise I cannot run the tests properly

**Comment by philbucher** [comment]
@msandre could you please in the meantime try this in your machine (since there the tests are working) and update if necessary the python-imports?
Thx

File: applications/HDF5Application/HDF5Application.py [hunk 1]
```diff
@@ -1,14 +1,8 @@
...
 application = KratosHDF5Application()
 application_name = "KratosHDF5Application"
 application_folder = "HDF5Application"
 
-# The following lines are common for all applications
-from .. import application_importer
-import inspect
-caller = inspect.stack()[1] # Information about the file that imported this, to check for unexpected imports
-application_importer.ImportApplication(application,application_name,application_folder,caller,__path__)
+KM._ImportApplicationAsModule(application, application_name, application_folder, __path__)
```
Comment (philbucher) [inline applications/HDF5Application/HDF5Application.py:8]: this is the actual change, others are only updating of imports

**Comment by philbucher** [comment]
This is ready to be merged
Tests in the App are running as well as the tests in Fluid & Structure

**Review by msandre — APPROVED** [review]
Thanks @philbucher :+1:

**Pull request merged**